src/wrappers/themis/wasm/src/secure_session.js

```diff
@@ -0,0 +1,413 @@
+// Copyright (c) 2019 Cossack Labs Limited
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file
+ * Themis Secure Session.
+ */
+
+const libthemis = require('./libthemis.js')
+const keygen = require('./secure_keygen.js')
+const errors = require('./themis_error.js')
+const utils = require('./utils.js')
+
+const subsystem = 'SecureSession'
```

Comment:
why subsystem? it's commonly used name?) looks like some module, not like name. can we rename to `subsystem_name` or `module_name`?

Reply:
Mmm... IIRC, I used this name because the documentation refers to Secure Cell, Secure Message, etc. collectively as [_cryptosystems_](https://docs.cossacklabs.com/pages/cryptosystems/#themis-cryptosystems). No idea why 'sub-' though. That's also the parameter name used in themis_error.js. 

Would `cryptosystem_name` be okay? I'll rename it here and in all other PRs as well.

Reply:
it's okay. `subsystem` looks like system library to work with some low-level api of some module, not like just name of currect module )